docs/reference/exception-classes.txt

```diff
@@ -67,10 +67,9 @@ MongoDB\\Exception\\UnexpectedValueException
    This exception is thrown when a command response from the server is
    malformed or not what the library expected. This exception means that an
    assertion in some operation, which abstracts a database command, has failed.
-   It may indicate a corrupted BSON response or bug in the server, driver, or
-   library.
+   It may indicate a corrupted BSON response or bug in the server or driver.
```

Comment:
Is driver referring to the extension, library or extension+library in this line?

Reply:
Any remaining references to "driver" should be taken to mean extension and/or library, where there is no need to differentiate.